addons/survey/models/survey_question.py

```diff
@@ -178,6 +181,12 @@ def _compute_save_as_email(self):
             if question.question_type != 'char_box' or not question.validation_email:
                 question.save_as_email = False
 
+    @api.depends('question_type')
+    def _compute_save_as_nickname(self):
+        for question in self:
+            if question.question_type != 'char_box':
```

Comment:
self.filtered

Reply:
Meh.
This is the same as `_compute_save_as_email`.
I personally think it's easier to read than a `filtered`.
It's also very common to iterate on all items in a compute method.